packages/storybook/stories/components/tag/tag.stories.ts

```diff
@@ -0,0 +1,175 @@
+import type { Meta, StoryObj } from '@storybook/web-components';
+import { defineCustomElement } from '@ovhcloud/ods-components/dist/components/ods-tag';
+import { ODS_ICON_NAMES, ODS_TAG_COLOR, ODS_TAG_COLORS, ODS_TAG_SHAPE, ODS_TAG_SHAPES, ODS_TAG_SIZE, ODS_TAG_SIZES, ODS_TAG_VARIANT, ODS_TAG_VARIANTS } from '@ovhcloud/ods-components';
+import { html } from 'lit-html';
+import { CONTROL_CATEGORY, orderControls } from '../../control';
+
+defineCustomElement();
+
+const meta: Meta = {
+  title: 'ODS Components/Content/Tag',
+  component: 'ods-tag',
+};
+
+export default meta;
+
+export const Demo: StoryObj = {
+  render: (arg) => html`
+<ods-tag class="my-tag-demo"
+         color="${arg.color}"
+         icon="${arg.icon}"
+         is-disabled="${arg.isDisabled}"
+         label="${arg.label}"
+         shape="${arg.shape}"
+         size="${arg.size}"
+         variant="${arg.variant}">
+</ods-tag>
+<style>
+  .my-tag-demo::part(tag) {
+    ${arg.customCss}
+  }
+</style>
+  `,
+  argTypes: orderControls({
+    color: {
+      table: {
+        category: CONTROL_CATEGORY.design,
+        defaultValue: { summary: ODS_TAG_COLOR.information },
+        type: { summary: ODS_TAG_COLORS }
+      },
+      control: { type: 'select' },
+      options: ODS_TAG_COLORS,
+    },
+    customCss: {
+      table: {
+        category: CONTROL_CATEGORY.design,
+        defaultValue: { summary: 'ø' },
+        type: { summary: 'string' }
+      },
+      control: 'text',
+      description: 'Set a custom style properties. Example: "background-color: #008000;"',
+    },
+    icon: {
+      table: {
+        category: CONTROL_CATEGORY.design,
+        type: { summary: ODS_ICON_NAMES }
```

Comment:
For the icon the list is very long, maybe a link to the icon doc should be better